### PR TITLE
Save window settings

### PR DESCRIPTION
**Pull request by arj03**
It now uses patch-settings :)

### CODE REVIEW

**Comment by arj03** [comment]
Right, namespacing is a good idea.

File: app/html/app.js [hunk 1]
```diff
@@ -56,6 +59,29 @@ exports.create = function (api) {
       addError(ev.error || ev)
     })
 
+    ////// TODO - extract this to keep patch-lite isolated from electron
+    const { getCurrentWebContents, getCurrentWindow } = electron.remote
+    window.addEventListener('resize', () => {
+      var wc = getCurrentWebContents()
+      wc && wc.getZoomFactor((zf) => {
```
Comment (mixmix) [inline app/html/app.js]: @arj03 I've set it up so the settings takes objects - I've changed it to use this for name-spacing

File: app/html/app.js [hunk 2]
```diff
@@ -56,6 +59,29 @@ exports.create = function (api) {
...
+          }
+        })
+      })
+    })
+
+    var zoomFactor = api.settings.sync.get('electron.zoomFactor')
+    if (zoomFactor)
+      getCurrentWebContents().setZoomFactor(zoomFactor)
+
+    var bounds = api.settings.sync.get('electron.windowBounds')
```
Comment (mixmix) [inline app/html/app.js]: here's what getting the settings out looks like (I think I mentioned in docs that it follows lodash/get style)

**Comment by mixmix** [comment]
works well, I can see why you're keen for this !

**Comment by mixmix** [comment]
Merged and published. Nice work @arj03 :fireworks: :pineapple:

**Pull request merged**